README.md

```diff
@@ -1,6 +1,8 @@
 
 
-# Microsoft Azure SDK for .NET
+## All PRs should be opened against master branch. This branch is being retired and will be deleted.
```

Comment:
We have already defaulted our repo to `master` branch, don't think we need this change. Appreciate the help